Documentation/gettingstarted/k8s-install-rke.rst

```diff
@@ -0,0 +1,85 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _rke_install:
+
+********************************************
+Installation Using Rancher Kubernetes Engine
+********************************************
+
+This guide walks you through installation of Cilium on `Rancher Kubernetes Engine <https://rancher.com/products/rke/>`_,
+a CNCF-certified Kubernetes distribution that runs entirely within Docker containers.
+RKE solves the common frustration of installation complexity with Kubernetes by
+removing most host dependencies and presenting a stable path for deployment,
+upgrades, and rollbacks.
+
+Install a Cluster Using RKE
+===========================
+
+The first step is to install a cluster based on the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_.
+When creating the cluster, make sure to `change the default network plugin <https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins/custom-network-plugin-example/>`_
+in the config.yaml file.
+
+Change:
+
+.. code-block:: yaml
+
+  network:
+    options:
+      flannel_backend_type: "vxlan"
+    plugin: "canal"
+
+To:
+
+.. code-block:: yaml
+
+  network:
+    plugin: none
+
+Deploy Cilium
+=============
+
+.. tabs::
+
+    .. group-tab:: Installation via Helm v3
+
+        Install Cilium via ``helm install``:
+
+        .. parsed-literal::
+
+           helm repo add cilium https://helm.cilium.io
+           helm repo update
+           helm install cilium |CHART_RELEASE| \\
+              --namespace $CILIUM_NAMESPACE
+
+    .. group-tab:: Installation via ``quick-install.yaml``
+
+        Install Cilium via the provided ``quick-install.yaml``:
+
+        .. parsed-literal::
+
+            kubectl apply -f |SCM_WEB|/install/kubernetes/quick-install.yaml
+
+
+    .. group-tab:: Installation via ``experimental-install.yaml``
+
+        Install Cilium via the provided ``experimental-install.yaml``:
```

Comment:
I wonder if it's worth having a sentence or two about what makes the "experimental" install "experimental" and why someone might want to choose it. Looks like `Documentation/gettingstarted/quick-install.rst` has similar steps to this, but also includes a warning that at least describes what this may involve. (Bonus points if we can figure out a way to share the same snippet across multiple places in the docs).

/cc @cilium/hubble who might be able to provide some extra context about this experimental-install YAML.

Reply:
I failed to find a good template that does this for all forms of installation. I'll create an issue to templatize this section taking what I've done here as a base example.

Reply:
Sounds good. Will you also add text that says what the experimental install is? Currently there's no context for users to understand why they should try to use or try to avoid this option.